tests/testthat/test-compiler-errors.R

```diff
@@ -209,20 +209,35 @@ test_that("Same argument passed to aes and geom is an error", {
   )
   expect_error({
       animint2dir(viz, open.browser=FALSE)
-  }, "Same argument cannot be passed to both aes and geom. Argument passed to both aes and geom: alpha")
+  }, "Same argument cannot be passed to both aes and geom. Argument passed to aes: alpha . Arguments passed to geom: alpha . The visual property needs only be defined in one place, so if it should be different for each rendered geom, but not depend on selection state, then it should be defined in aes; but if the property should depend on the selection state then it should be defined in geom")
 })
 
 test_that("alpha and alpha_off passed to aes and geom is an error", {
   scatter <- ggplot()+geom_path(aes(
     x=life.expectancy, 
     y=fertility.rate, 
     color=region,
-    group=country, alpha=population),alpha_off=0.5,
+    group=country, alpha=population),clickSelects="country", alpha_off=0.5,
     data=WorldBank)
   viz <- list(
     plot = scatter
   )
   expect_error({
     animint2dir(viz, open.browser=FALSE)
-  }, "Same argument cannot be passed to both aes and geom. Argument passed to both aes and geom: alpha")
+  }, "Same argument cannot be passed to both aes and geom. Argument passed to aes: alpha . Arguments passed to geom: alpha_off . The visual property needs only be defined in one place, so if it should be different for each rendered geom, but not depend on selection state, then it should be defined in aes; but if the property should depend on the selection state then it should be defined in geom")
```

Comment:
this is better but still "argument" may be confusing, maybe change to "same visual property cannot be defined in both aes and geom" ?
Also please remove space before period: "alpha ." -> "alpha." try paste0 instead of paste.

Reply:
Thanks, I rewrote the error message with the updated text and removed space before period

Reply:
great